Automattic/wp-calypso
# Reader site titles: get rid of "Site Title"

(Site Title) is ugly.  Lets fallback to the url in the case where the user never changed from the default

To test:
- look at this url before and after: http://calypso.localhost:3000/read/search?isSuggestion=1&q=autoblog618&show=Sites

you should see "Site Title" before, url after
